rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -44,6 +44,22 @@ import com.normation.inventory.domain.NodeId
 
 
 /**
+ * Remark on above: RuleExpectedReport does not
+ * map to anything usefull today. We would like to
+ * have something like:
+ */
+case class RuleNodeExpectedReports(
```

Comment:
I DOES have link with node, it's the expected reports for a given node - or at least the intent is that, contrary to expected reports for a rule, and that's the reason why there is no nodeId/configId on it. 
The name was chosen to match the corresponding compliance report (RuleNodeStatusReport). 

I believe we can't remoce RuleExpectedReport for now because of some method that still use it (for plugin?). I could be renamed (DeprecatedExpectedReportForRule ?)